_posts/2016-10-31-pulp-3.0-mvp.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Pulp Celerybeat and the Pulp 3.0 Minimum Viable Product (MVP)
+date: 2016-08-18T18:28:49+00:00
+layout: post
+permalink: /2016/08/18/sprint-demo-6-august-18-2016/
+author: Dennis Kliban
+tags:
+  - MVP
```

Comment:
We have to specifically enable tags, we don't have one for MVP so this should be removed probably.